### PR TITLE
Adding QuizUp to adopters list

### PR DESCRIPTION
**Pull request by arnarthor**

### CODE REVIEW

**Comment by vkostyukov** [comment]
Thank you @arnarthor! Some context from the [Finch Gitter channel](https://gitter.im/finagle/finch?at=56159d7476d984a35875c13a).

File: ADOPTERS.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ URLs if possible) and [submit a pull request][1]!
   * ["How we make pins more useful"][3]
 * [Quanttus](http://www.Quanttus.com/)
 * [Qubit](http://www.qubitproducts.com/)
+* [QuizUp] (http://www.quizup.com)
```
Comment (mosesn) [inline ADOPTERS.md]: for consistency, could you https and add a trailing slash?

Comment (arnarthor) [inline ADOPTERS.md]: Sure thing

Comment (arnarthor) [inline ADOPTERS.md]: Fixed in  3870599

**Comment by vkostyukov** [comment]
I'm pulling this internally. Will close this PR once the change make it to the `develop` branch.

**Comment by vkostyukov** [comment]
This has been merged internally. Thanks @arnarthor!

**Pull request closed without merging**